CMakeLists.txt

```diff
@@ -320,22 +318,19 @@ if(${pluginConf} IS_NEWER_THAN ${pluginCode})
 endif(${pluginConf} IS_NEWER_THAN ${pluginCode})
 
 # create file with files and dirs for source tarball
-#if(${pluginConf} IS_NEWER_THAN ${srcTarball})
-  set(dirList "CMakeLists.txt" ${versionFile} ${pluginConf} "doc" "examples" ${libList})
-  if (${AMGX_INCLUDE_EXTERNAL})
-    set(dirList ${dirList} "external")
-  endif (${AMGX_INCLUDE_EXTERNAL})
-  set(dirList ${dirList} "tests")
-  string(REPLACE "/" " " dirliststr ${CMAKE_SOURCE_DIR})
-  string(REGEX MATCHALL "([^\ ]+\ |[^\ ]+$)" dirlist "${dirliststr}")
-  list(GET dirlist -1 branch)
-  #message (STATUS "working with branch = " ${branch})
-  file(WRITE ${srcTarball} "")
-  foreach( dir ${dirList} )
-    file(TO_NATIVE_PATH "${branch}/${dir}" nativePath)
-    file(APPEND ${srcTarball} "${nativePath}\n")
-  endforeach(dir)
-#endif(${pluginConf} IS_NEWER_THAN ${srcTarball})
+set(dirList "CMakeLists.txt" ${versionFile} ${pluginConf} "doc" "examples" ${libList})
+if (${AMGX_INCLUDE_EXTERNAL})
+  set(dirList ${dirList} "external")
+endif (${AMGX_INCLUDE_EXTERNAL})
+set(dirList ${dirList} "tests")
+string(REPLACE "/" " " dirliststr ${CMAKE_SOURCE_DIR})
+string(REGEX MATCHALL "([^\ ]+\ |[^\ ]+$)" dirlist "${dirliststr}")
+list(GET dirlist -1 branch)
+#message (STATUS "working with branch = " ${branch})
+foreach( dir ${dirList} )
+  file(TO_NATIVE_PATH "${branch}/${dir}" nativePath)
+  file(APPEND ${srcTarball} "${nativePath}\n")
```

Comment:
Still working with srcTarball here, should we be?

Reply:
No, I don't think so., but I'd prefer to fix this in a separate PR that focus on cleaning up the CMake build system if that makes sense.

Reply:
OK fine by me